R/getmask.R

```diff
@@ -0,0 +1,72 @@
+#' Get mask
+#'
+#' @description Build a mask with the intersection of the topography and the spatial inventory.
+#'
+#' @param inventory Input inventory (see the inputs formats and metadata in the
+#'   vignette) (data.frame)
```

Comment:
is it the raw or prepared inventory that is required here ? I guess it's the raw and it that true, than nothing to change. If it's the prepared, we have to say it explicitly.

Reply:
This is the raw one.